requirements.txt

```diff
@@ -1,3 +1,3 @@
-awscli>=1.10.16
+awscli>=1.10.48
```

Comment:
It is not used by any of the containers or automation, it is only used by ourselves for local development.

The awscli container has a good version... On the other hand, it would make sense to use the same version than awscli containers :-m